requirements.txt

```diff
@@ -3,8 +3,9 @@ mock==1.0.1
 Delorean==0.4.5
 botocore==1.1.2
 flake8==2.4.1
-nose==1.3.7
 requests==2.7.0
 six==1.8.0
 tox==2.1.1
 python-coveralls==2.5.0
+pytest==3.0.7
+pytest-cov==2.4.0
```

Comment:
since these are not production requirements, it would be better to put them into a separate file, eg.:

requirements-dev.txt
```
-rrequirements.txt
pytest==3.0.7
pytest-cov==2.4.0
```

...and then use this file below in `tox.ini` as a dependecies file